automation/release.sh

```diff
@@ -72,9 +72,7 @@ function update_github_release() {
 
 function upload_testing_manifests() {
     # replaces periodic-kubevirt-update-release-x.y-testing-manifests periodics
-    if [ "$(gsutil ls 'gs://kubevirt-prow/devel/release/kubevirt/kubevirt/'|grep $DOCKER_TAG|wc -l)" -gt 0 ]; then
```

Comment:
Doid you consider :https://cloud.google.com/storage/docs/gsutil/commands/stat?

Seems like `gsutil -q stat gs://some-bucket/some-subdir/` seems to be extra there for scripting.

Reply:
I remember having considered using it a while ago, don't remember why I didn't. Will look into this.

Reply:
Now I remember: it exits with 1 for non existing objects. Therefore I initially went with the `gsutil ls ... | grep` approach. So if we use stat we would also have to ignore errors, which IMHO makes no difference to just trying to delete it blindly. WDYT?

Reply:
ack